src/cameras/map/index.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { Grid } from '@material-ui/core'
+import GoogleMapReact from 'google-map-react'
+
+import { useSettings } from 'settings/hooks/use-settings'
+import { useStyles } from 'cameras/map/style'
+import { useItems } from 'cameras/list/items/hooks/use-item'
+import { Marker } from 'cameras/map/marker'
+
+export const Map = () => <Grid container className={useStyles().grid}>
+  <GoogleMapReact defaultCenter={{ lat: 45.79, lng: 24.13 }} defaultZoom={15}
```

Comment:
Shouldn't this `defaultCenter` prop be configurable? After all, if Nerds moves to Tajikistan, should we have to redeploy the app? :smile:

Reply:
Done!

Reply:
![image](https://user-images.githubusercontent.com/23569938/129866329-ed31b4fd-7ba5-4255-8fb9-a38a7fd25562.png)
I don't mean to be rude, but this code turned hideous. Let's at least extract these props into a separate hook.

Reply:
Done!